packages/nx-playwright/src/generators/project/lib/add-linting.ts

```diff
@@ -26,13 +26,25 @@ export async function addLinting(
       host,
       joinPathFragments(options.projectRoot, '.eslintrc.json'),
       (json: EslintIgnoreFile) => {
-        const overrides = json.overrides.map(({ files, rules }) =>
+        const baseOverrides = json.overrides.map(({ files, rules }) =>
           files.includes('*.ts')
             ? { files, rules: { 'jest/no-done-callback': 'off' } }
             : { files, rules },
         );
 
-        return { ...json, overrides, extends: [...json.extends] };
+        return {
+          ...json,
+          overrides: options.includeAxe
+            ? [
+                ...baseOverrides,
+                {
+                  files: ['**/axe-tests/axe-tests.spec.ts'],
+                  rules: { 'jest/expect-expect': 'off' },
+                },
+              ]
+            : baseOverrides,
+          extends: [...json.extends],
```

Comment:
Will this duplicate this override if the override was already pressent?

Reply:
As far as I know it shouldn't because this generator is creating a new `.eslintrc.json` file for the app. It is not writing over an existing one. But please correct me if I'm wrong.